test/integration.js

```diff
@@ -42,6 +42,9 @@ test('retrieval check for our CID', async () => {
   assertProp('protocol', 'http')
   assertProp('timeout', false)
   assertProp('statusCode', 200)
+  // FIXME - frisbii.fly.io doesn't support HEAD requests yet
+  // https://github.com/CheckerNetwork/frisbii-on-fly/issues/3
+  // assertProp('headStatusCode', 200)
```

Comment:
What is the error code returned by Frisbii? Can we keep the assertion but check the error code currently returned by our Frisbii? My goal is to ensure that new measurements include `headStatusCode` field (i.e. all implementation pieces are correctly integrated together in Spack Checker).

From the point of checking HEAD requests, the checker does the right thing when it reports that Frisbii does not support HEAD requests - that's the signal we want to get!

```suggestion
  // Note: frisbii.fly.io doesn't support HEAD requests yet
  // https://github.com/CheckerNetwork/frisbii-on-fly/issues/3
  assertProp('headStatusCode', 405)
```

Reply:
```
❯ curl -I https://frisbii.fly.dev/ipfs/bafkreih25dih6ug3xtj73vswccw423b56ilrwmnos4cbwhrceudopdp5sq\?dag-scope\=block
HTTP/2 405
allow: GET
date: Wed, 19 Feb 2025 09:46:59 GMT
content-length: 18
content-type: text/plain; charset=utf-8
server: Fly/eb1a25638 (2025-02-13)
via: 2 fly.io
fly-request-id: 01JMESMZVY238QQQTH0GKTMYD3-fra
```